src/OpenTelemetry/Trace/ExceptionProcessor.cs

```diff
@@ -75,7 +75,11 @@ public override void OnEnd(Activity activity)
 
             if (snapshot != pointers)
             {
+                // TODO: Remove this when SetStatus is deprecated
```

Comment:
The old API call is not needed.

Reply:
That would be a breaking change for exporters still checking tags for status.

Reply:
The exporters have already been updated to also look for `ActivityStatusCode`.

Reply:
Yes. I think we are stuck with this till 2.0 release then. This does incur some perf cost ☹️